k8s.gcr.io/images/k8s-staging-build-image/images.yaml

```diff
@@ -392,22 +394,32 @@
     "sha256:ff6e2f3683e7d284674ed18341fc898060204e8c43c9b477e13c6f7faf3e66d4": ["buster-v2.0.0"]
 - name: go-runner-amd64
   dmap:
+    "sha256:1ca263fc76083cc779310b665e6f8c5b37078dc7935d9b1ef084ca27ac3c6847": ["v2.3.1-go1.17.7-bullseye.0"]
+    "sha256:2e6c3bd28ae0aa3cb25171888c671b00fa48999f744d30abacba9bd1f22912d9": ["v2.3.1-go1.16.14-buster.0"]
```

Comment:
we can remove those, the existing ones pass thru, but we don't need to promote those